warnings.h

```diff
@@ -20,6 +20,7 @@ class Warnings
     void setup(Ui::MainWindow*);
     void on(warnings warning);
     void off(warnings warning);
+    void CheckRange();
```

Comment:
You can remove this function declaration since it was declared and defined in the battery class